geometry_msgs/Pose2D pose
float64 vl
float64 vr
float64 gx
float64 gy
float64 v_pref
float64 radius
